site/en/docs/webstore/i18n/index.md

```diff
@@ -1,132 +1,124 @@
+| ja          | Japanese                              |
+| kn          | Kannada                               |
+| ko          | Korean                                |
+| lt          | Lithuanian                            |
+| lv          | Latvian                               |
+| ml          | Malayalam                             |
+| mr          | Marathi                               |
+| ms          | Malay                                 |
+| nl          | Dutch                                 |
+| no          | Norwegian                             |
+| pl          | Polish                                |
+| pt_BR       | Portuguese (Brazil)                   |
+| pt_PT       | Portuguese (Portugal)                 |
+| ro          | Romanian                              |
+| ru          | Russian                               |
+| sk          | Slovak                                |
+| sl          | Slovenian                             |
+| sr          | Serbian                               |
+| sv          | Swedish                               |
+| sw          | Swahili                               |
+| ta          | Tamil                                 |
+| te          | Telugu                                |
+| th          | Thai                                  |
+| tr          | Turkish                               |
+| uk          | Ukrainian                             |
+| vi          | Vietnamese                            |
+| zh_CN       | Chinese (China)                       |
+| zh_TW       | Chinese (Taiwan)                      |
+
+## Localize your extension listing
+
+To localize your extension's listing, you first localize the name and description. Then, after uploading
+your localized extension, you can edit your extension's listing to specify localized versions of your extension's
 detailed description and screenshots.
 
-### Name and description
+### Name and description {: #localize-manifest }
+
+You localize the [name][name] and [description][description] by modifying `manifest.json` and providing a `_locales`
+directory in your extension's ZIP file. Specifically, you need to:
 
-You localize the name and description by modifying `manifest.json` and providing a `_locales`
-directory in your app's ZIP file. Specifically, you need to:
+- Modify the `"name"` and `"description"` manifest fields.
+- Add a `"default_locale"` field.
+- Provide at least one `_locales/LOCALE_CODE/messages.json` file. Replace
+  [*LOCALE_CODE*][supported-locales] with a supported locale; each `messages.json` file contains the strings for one
+  locale.
 
-- Modify the "name" and "description" manifest fields
-- Add a "default_locale" field
-- Provide at least one `_locales/_locale_/messages.json` file, where _`locale`_ is a [locale
-  code][12]; each `messages.json` file contains the strings for one locale
+{% Aside %}
+The name should not exceed 45 characters and the description 132 characters.
+{% endAside %}
```

Comment:
```suggestion
{% Aside %}

The name should not exceed 45 characters and the description should not exceed 132 characters.

{% endAside %}
```

First, this is a great callout. While Chrome doesn't have length requirements on these fields, CWS does.

My suggestion adds space within the aside to make formatting easier for automated tooling and tweaks the phrasing on this sentence. While the current copy is grammatically sound, I think it would be safer to err on the side of explicitness and be a little repetitive. This may also help make the point clearer to non-native English speakers.